models/webhook.go

```diff
@@ -753,7 +756,21 @@ func (t *HookTask) deliver() {
 	t.IsDelivered = true
 
 	timeout := time.Duration(setting.Webhook.DeliverTimeout) * time.Second
-	req := httplib.Post(t.URL).SetTimeout(timeout, timeout).
+
+	var req *httplib.Request
+	if t.HTTPMethod == "post" {
```

Comment:
Could you use `http.MethodGet` here?

Reply:
`http.MethodGet` is uppercase "GET", unfortunately.

Reply:
perhaps storing the method as uppercase in DB would be better, that way constants can be matched.